docs/navigation.html

```diff
@@ -185,12 +185,12 @@
 
 <ul><div>The Store</div>
   <li {% if page.path == 'Store.md' %} class="active" {% endif %}><a class="nav-link" href="./Store.html">Introduction</a></li>
-  <li {% if page.path == 'LocalesMenuButton.md' %} class="active" {% endif %}><a class="nav-link" href="./LocalesMenuButton.html"><code>&lt;LocalesMenuButton&gt;</code></a></li>
```

Comment:
You removed the `LocalesMenuButton` link. Is this ok?

Reply:
Yes, because it's already in the i18n section